### PR TITLE
OSSMDOC-638: IOR should copy labels from Gateway to Route

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.9-4.12

Issue: https://issues.redhat.com/browse/OSSMDOC-638

Link to docs preview: http://file.bos.redhat.com/tokeefe/OSSMDOC-638/service_mesh/v2x/ossm-traffic-manage.html#ossm-auto-route-annotations_routing-traffic

Additional information:

Dev review - Jonh 
QE review - Sunil

### CODE REVIEW

**Comment by rh-tokeefe** [comment]
@jwendell please review. Thanks!

**Review by jwendell — APPROVED** [review]

**Comment by rh-tokeefe** [comment]
@skondar please review when you have time. Thanks!

**Comment by rh-tokeefe** [comment]
@skondar please provide QE review. Thanks!

**Comment by skondkar** [comment]
LGTM

**Comment by rh-tokeefe** [comment]
@JStickler this is ready for peer review.

File: modules/ossm-auto-route-annotations.adoc [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
 
-[id="ossm-auto-route-annotations_{context}"]
-= {SMProductName} route annotations
+[id="ossm-auto-route-labels-and-annotations_{context}"]
+= {SMProductName} route labels and annotations
 
-Sometimes specific annotations are needed in an OpenShift Route.
+Sometimes specific labels or annotations are needed in an OpenShift Route.
 ifdef::openshift-enterprise[]
 For example, some advanced features in OpenShift Routes are managed via xref:../../networking/routes/route-configuration.adoc[special annotations].
```
Comment (JStickler) [inline modules/ossm-auto-route-annotations.adoc:10]: XREFs inside modules are not allowed.  This is one of the XREFs that is listed on [OSSMDOC-593](https://issues.redhat.com/browse/OSSMDOC-593) that needs to be removed.

**Review by JStickler — COMMENTED** [review]
Please remove the XREF and revise appropriately.

File: modules/ossm-auto-route-annotations.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ossm-auto-route-annotations.adoc]: @JStickler the xref has been updated. I also added a comment in 593 just as a reminder. 

Preview build has been updated and PR has been updated.

Comment (rh-tokeefe) [reply modules/ossm-auto-route-annotations.adoc]: @JStickler if this looks good after the revision, then we can merge.

File: service_mesh/v2x/ossm-traffic-manage.adoc [hunk 2]
```diff
@@ -42,6 +42,11 @@ include::modules/ossm-auto-route.adoc[leveloffset=+2]
 
 include::modules/ossm-auto-route-annotations.adoc[leveloffset=+2]
 
+ifdef::openshift-enterprise[]
+.Additional resources
+* xref:../../networking/routes/route-configuration.adoc[Special annotations].
```
Comment (JStickler) [inline service_mesh/v2x/ossm-traffic-manage.adoc:47]: Did you want to link to the Assembly or to the route-specific annotations topic in this assembly?

https://docs.openshift.com/container-platform/4.11/networking/routes/route-configuration.html#nw-route-specific-annotations_route-configuration 

A tip for creating the XREF is to copy the URL for the specific heading you want, and then replace the first part of the URL with the ../../ for the relative path, and the HTML with adoc.  So the XREF for the heading above would be:

`xref:../../networking/routes/route-configuration.adoc#nw-route-specific-annotations_route-configuration[HEADING TEXT]`

Comment (rh-tokeefe) [reply service_mesh/v2x/ossm-traffic-manage.adoc]: Thanks for that tip @JStickler! The PR and preview build have been updated. If this looks good, we can merge it.

**Review by JStickler — APPROVED** [review]
LGTM

**Comment by rh-tokeefe** [comment]
@JStickler please merge if you have time.

**Pull request merged**